locales/en.json

```diff
@@ -286,10 +285,18 @@
     }
   },
   "RequestScreen": {
-    "bluetoothDenied": "Please enable Bluetooth to be able to request Card",
+    "bluetoothDenied": "Please enable Bluetooth to be able to request {{vcLabel}}",
```

Comment:
replace the vclabel placeholder here as well.

Reply:
`vcLabel` is being used in multiple places, should this be remove for all specific translation strings or should I just change `vcLabel` for all language files to `Card`? The latter approach seems easier.

Reply:
@vijay151096 I've made the suggested changes to the `locales/<lang>.json` files post doing a rebase against develop. Please take a look.